deno/UPDATING.md

```diff
@@ -0,0 +1,5 @@
+# How-To Update Bindings When gtfs-realtime.proto Changes
+
+## Regenerate the language binding source from gtfs-realtime.proto.
+
+1. make nodejs/gtfs-realtime.js and edit as ES module
```

Comment:
Due to the overhead of maintaining bindings in multiple languages, we've been shying away from adopting any new languages where the update process can't be automated. It looks like this updating process requires hand-editing to update the bindings? More details here are needed.

@maximearmstrong would need to weigh in on MobilityData's willingness to take on the additional responsibility of manually updating these bindings going forward.

Related - we should update the ["Other languages" section](https://github.com/MobilityData/gtfs-realtime-bindings#other-languages) of the README to more specifically itemize the requirements for new binding language contributions (e.g., supported by official PB tooling, process to update and publish release can be automated).